app/src/main/res/layout/dialog_critical_app_update.xml

```diff
@@ -92,13 +92,12 @@
     app:layout_constraintTop_toBottomOf="@+id/helpSectionTitle" />
 
   <com.google.android.material.button.MaterialButton
+    style="@style/Widget.Simple.Button"
```

Comment:
The elevation is still not working. I don't think it's the style that's causing the issue, this is applied by default to all buttons. The issue seems to be because of the parent padding, the shadow seems to be getting clipped. Can you take a look at that once, please?

Reply:
We can get rid of the style and elevation from this button if that's the case.